DataStructureMemories/DataStructureMemories/DataStructureTableViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  DataStructureTableViewController.swift
+//  DataStructureMemories
+//
+//  Created by Alina Zaitseva on 4/12/18.
+//  Copyright © 2018 Alina Zaitseva. All rights reserved.
+//
+
+import UIKit
+
+class DataStructureTableViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    let dataStructureManipulatios = DataCaprureMemoryStructModel()
```

Comment:
Não falo português, could you use more "English" names?

Reply:
Sure)